include/swift/Runtime/Config.h

```diff
@@ -140,6 +140,10 @@
 // been integrated into clang and llvm.
 #define SWIFT_RT_USE_RegisterPreservingCC 0
 
+#if !defined(__has_attribute)
+#define __has_attribute(x) 0
```

Comment:
Yeah true. What I've done is delegated that kind of stuff to "LLVM/Support/Compiler.h" - see the 3rd commit